### ISSUE TITLE
NVDA (Version 2023.2.0.29051) Does Not Announce Code Suggestions in Eclipse IDE for Java Developers - 2023-06

### ISSUE THREAD

**Issue by rezabakhshilaktasaraei** [nvaccess/nvda#15413]
### Steps to reproduce:
1.
Launch Eclipse IDE for Java Developers (2023-06).
2.
Open a Java file.
3.
Place the cursor where code suggestions are expected (e.g., within a method).
4.
Press Ctrl + Space.

### Actual behavior:

In the latest NVDA version (2023.2.0.29051), there is no announcement or indication of code suggestions when pressing Ctrl + Space in Eclipse IDE.

### Expected behavior:

NVDA (Version 2023.2.0.29051) should announce and provide a list of code suggestions, as it did in the previous NVDA version 2023.1 (2023.1.0.27913).
#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2023.2 (2023.2.0.29051)

#### Windows version:
10 & 11
#### Name and version of other software in use when reproducing the issue:
Eclipse IDE for Java Developers - 2023-06

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2023.1 (2023.1.0.27913)
I don't have this issue in this version 2023.1 (2023.1.0.27913).

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by lukaszgo1** [comment]
I cannot reproduce this using:
Eclipse IDE for Java Developers (includes Incubating components)

Version: 2023-06 (4.28.0)
Build id: 20230608-1333


Would you be able to set your level  log to debug, reproduce the problem in both 2023.1 and 2023.2, and upload the debug logs to this issue?

**Comment by lukaszgo1** [comment]
@rezabakhshilaktasaraei  Any updates here?

**Comment by rezabakhshilaktasaraei** [comment]
How do I set in debug?

**Comment by lukaszgo1** [comment]
NVDA+q, select 'Restart with debug logging enabled'.

**Comment by rezabakhshilaktasaraei** [comment]
Thank you. I found it. I will send you whenever the issue occurs for me. Unfortunately, this issue is not stable.

**Comment by thgcode** [comment]
When you test it again, please also check the order of Eclipse's autocomplete suggestions dialog on object navigation. On the current code, it should be a child of the desktop object and right of it should have a button. Maybe this logic needs to be changed when the machine places something else after the dialog...

**Comment by Adriani90** [comment]
No updates since more than two years upon request, closing as abandoned. If you are still having this issue, please comment and we can reopen.
